src/components/ProfileTop.vue

```diff
@@ -139,6 +137,8 @@ export default {
 .director {
   text-decoration: underline;
   font-size: 12px;
+  line-height: 16.37px;
+  font-family: 'nunito-sans-regular-400 - latin';
```

Comment:
boyle bi font family yok, font-family'i component'lerde kullanmamiza da gerek yok cunku styles.scss'de @font-face ile set edildi.

Reply:
set edildi ama 3 çeşit var . Nerde hangi kalınlıktakini kullanacağını anlaması için farklı isimler vermemeiz gerektiğini düşündük.

Reply:
ustte yaptigim yoruma bkz

Reply:
eger bi yorumla isiniz bittiyse onun altindaki resolve conversation butonuna basabilirsiniz

Reply:
anladım thnx